### PR TITLE
Fixed laundry.py code for new url

### PR DESCRIPTION
**Pull request by jkruse7**
I fixed the code for laundry.py so it works with the new url, and correctly reads the machines for the different buildings.

### CODE REVIEW

File: pittapi/laundry.py [hunk 1]
```diff
@@ -84,41 +151,158 @@ def get_status_detailed(building_name: str) -> List[Dict[str, Union[str, int]]]:
...
         -> SUTH_EAST
         -> SUTH_WEST
     """
+    laundry_soup = _get_laundry_soup(building_name)
+
+    total_washers = 0
+    total_dryers = 0
+    free_washers = 0
+    free_dryers = 0
+    in_use_tag = "In Use"
```
Comment (RitwikGupta) [inline pittapi/laundry.py]: Can you turn this into an enum and use that instead?

File: pittapi/laundry.py [hunk 2]
```diff
@@ -62,19 +59,89 @@ def get_status_simple(building_name: str) -> Dict[str, str]:
...
+    total_dryers = 0
+    free_washers = 0
+    free_dryers = 0
+
+    machine_dict = parse_soup(laundry_soup)
+    # each building has a different system for washer vs dryer
+    if(building_name == 'SUTH_EAST'):
+        for machine in machine_dict.keys():
+            num = int(machine)
+            if num >= 5 and num <= 14:
```
Comment (RitwikGupta) [inline pittapi/laundry.py:84]: What was the methodology to figure these indices out? out of curiosity.

**Review by RitwikGupta — CHANGES_REQUESTED** [review]

File: pittapi/laundry.py (hunk 1 above)
Comment (jkruse7) [reply pittapi/laundry.py]: Just made the changes so should show up in this pr. Let me know if anything else needs fixing!

File: pittapi/laundry.py (hunk 2 above)
Comment (jkruse7) [reply pittapi/laundry.py:84]: For towers (and I think Sutherland?) there was actually a system with like odds and evens being dryers and washers, but for the smaller buildings it was kind of just however they wanted to arrange it, so I had to organize it manually by going through the site and reading off which ids matched to a washer vs. dryer. That code is me just organizing those ids into their right categories. Not great, but I couldn't figure any other way to do it.

**Comment by tianyizheng02** [comment]
Closing this PR because it's 2 years old, and `laundry.py` has since diverged too much from this PR

**Pull request closed without merging**